server/index.js

```diff
@@ -1,28 +1,34 @@
-const path = require('path');
+// const path = require('path');
 const express = require('express');
-const query = require('./database/queries');
+const query = require('./database/queries.js');
 
 const app = express();
 const port = 3000;
 
+app.use(express.json());
+app.use(express.static(__dirname + '/../client/dist'))
+
 app.get('/api/house/:id', (req, res) => {
-  query.getHouseInfoByID(req.params.id, (err, data) => {
+  console.log(req.params.id, 'req in app.get');
+  query.getHouseInfoByHostelID(req.params.id, (err, data) => {
     if (err) {
       res.sendStatus(500);
     } else {
+      console.log(res, 'res in app.get');
+      console.log(data, 'data in app.get');
       res.send(data);
     }
   });
 });
 
 app.get('api/house/:id/description', (req, res) => {
-  query.getHouseDescription(req.params.id, (err, data) => {
-    if (err) {
-      res.sendStatus(500);
-    } else {
-      res.send(data);
-    }
-  });
+  query.getHouseDescription(req.params.id)
+    .then((result) => {
+      res.status(201).json(result);
+    })
+    .catch((err) => {
+      res.status(500).send(err);
+    });
 });
```

Comment:
Unless you are defining the errors yourself, I would not send back the err object you receive the database. You want to limit the amount of knowledge someone has about your backend as much as possible and database errors give a lot. You could simply send a string back that is short and to the point. "Error getting description from the database"